docs/CNV/howtogcnv.md

```diff
@@ -0,0 +1,707 @@
+
+Here, SEG track deletions range from light blue to blue (CN1-CN0) and amplifications from salmon to red (CN3-CN5). As the twelve regions show, the case and cohort mode give highly concordant results to each other. Towards the truthset, by eye it appears they are missing three of the calls in the regions. Turning the sensitivity knob up gives better concordance with truthset calls, as gCNV then calls two of the three missing calls. Both the default and sensitive parameter runs pick up additional events not in the truthset.
+
+<center>[<img src="images-gcnv/gcnv_igv_three_modes_SIRPB1_20190321_sooheelee.png" align="center" width="700" />](images-gcnv/gcnv_igv_three_modes_SIRPB1_20190321_sooheelee.png)</center>
+
+If we focus on the first region, we see the default and sensitive parameter runs give differently sized events such that the functional impact on SIRPB1 expression is likely different. [Section 7](#7) discusses this further.  
+
+---
+
+<a name="6.1"></a>
+### 6.1 How do I make SEG format files from gCNV results?
+
+Here is one approach to convert gCNV VCF results to SEG format data. These command works in Unix. For Linux, modification will be necessary, e.g. `zcat` replaces `gzcat`.
+
+**[1] VariantsToTable to subset and columnize annotations**
+
+```
+gatk VariantsToTable \
+-V genotyped-segments-case-twelve-vs-cohort23.vcf.gz \
+-F CHROM -F POS -F END -GF NP -GF CN \
+-O genotyped-segments-case-twelve-vs-cohort23.table.txt
+```
+
+**[2] Unix shell commands to convert to SEG format data**
+
+This involves adding a first column with the sample name.
+
+```
+sampleName=$(gzcat genotyped-segments-case-twelve-vs-cohort23.vcf.gz | grep -v '##' | head -n1 | cut -f10)
+awk -v sampleName=$sampleName 'BEGIN {FS=OFS="\t"} {print sampleName, $0}' genotyped-segments-case-twelve-vs-cohort23.table.txt > ${i}.seg; head ${i}.seg
+```
+
+**Combine [1] and [2] into a for-loop to process multiple gCNV VCFs into SEGs**
+
+
+```
+for i in `ls genotyped-segments*.vcf.gz`; 
+do sampleName=$(gzcat $i | grep -v '##' | head -n1 | cut -f10); 
+gatk4100 VariantsToTable -V ${i} -F CHROM -F POS -F END -GF NP -GF CN -verbosity ERROR -O ${i}.table.txt; 
+awk -v sampleName=$sampleName 'BEGIN {FS=OFS="\t"} {print sampleName, $0}' ${i}.table.txt > ${i}.seg; head ${i}.seg; 
+done
+```
+
+<a name="7"></a>
+[back to top](#top)
+
+---
+## 7.	Some additional discussion
+
+If research aims focus on rare gCNV events only, e.g. non-multialleleic CNV singletons for a sample compared to a cohort, it is worth looking into the [GATK4 ModelSegments CNV workflow](https://software.broadinstitute.org/gatk/documentation/article?id=11682), which is sensitive to fractional changes and runs amazingly quickly. Note the ModelSegments CNV workflow is designed for somatic CNA detection. Below is one possible outcome from running ModelSegments CNV on NA19017 against a PoN made from the remaining 23 samples.
```

Comment:
OK to show this, but we should probably blacklist out the noisy (presumably centromeric) region.

Reply:
I had to move section 7 to its own article (likely blog post). I will work on omitting the noisy centromeric region.

Reply:
I tweaked the parameters a bit and so if you have some recommended parameters for this particular purpose, I would be happy to showcase them.